receiver/awscontainerinsightreceiver/internal/cadvisor/cadvisor_nolinux.go

```diff
@@ -0,0 +1,40 @@
+// Copyright  OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !linux
+
+package cadvisor
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/host"
+)
+
+// cadvisor doesn't support windows, define the dummy functions
+
+// Cadvisor is a dummy struct for windows
```

Comment:
Let's use term "no-op" instead of "dummy"

Reply:
Sure. I will make the change in later PRs. Thanks for reviewing it!